### DISCUSSION TITLE
Unable to monitor complete DB instance

### DISCUSSION THREAD

**Discussion by aakashm07** [cybertec-postgresql/pgwatch2#656] [Q&A]
Hi Team,

     I implemented pgwatch2 using docker containers. 

# docker run -d -p 3000:3000 -p 8080:8080 -e PW2_TESTDB=true --name pw2 cybertec/pgwatch2:1.10.0

It is working fine. But I am having 3 database clusters each having 2-3 non-template databases. Even though I left the database box empty in the admin panel, still it is not giving metrics for all the DBs. only showing the metrics for Postgres database. Kindly suggest if I am missing anything. Thanks.

Expected like it will show nodeA_db1, nodeA_db2,nodeA_db3 . NodeB_db1, NodeB_db2 and so on. But it is giving database specific.

<img width="1439" alt="Screenshot 2023-06-21 at 3 31 51 PM" src="https://github.com/cybertec-postgresql/pgwatch2/assets/45911698/ffa18b4a-d158-4e49-8da3-25a51b492bbd">
<img width="1440" alt="Screenshot 2023-06-21 at 3 33 10 PM" src="https://github.com/cybertec-postgresql/pgwatch2/assets/45911698/80a5f286-26bb-46e8-8614-61ba0c0196ae">

**Comment by pashagolub** [comment]
Hello.

When using the Web UI and the “DB name” field is left empty, there’s as a **one time operation** where all non-template DB names are fetched, prefixed with “Unique name” field value and added to monitoring **(if not already monitored)**. Internally monitoring always happens “per DB” not “per cluster” though.

I can see `prod1` which is kind of prefixed with your unique name `prod`. Would you, please, provide us with the list of databases you want to monitor?

**Reply by aakashm07** [reply]
my db Ip is 127.0.0.1 and port 5433 -> prod . Having 3 databases(X,Y,postgres) inside this db including postgres default database. 

another DB ip is 127.0.0.1 and port 5432 -> prod1. Having 3 databases(A,B,postgres) inside this db including postgres default database. 

Both are configured, but only showing default postgres database size and its tables. If I am giving particular database(X) under prod(unique name), it is showing the metrics.
